defaults/config.js

```diff
@@ -368,4 +368,11 @@ module.exports = {
 	// @default  false
 	//
 	debug: false,
+
+	// Enables logging raw IRC messages into each server window.
+	//
+	// @type     boolean
+	// @default  false
+	//
+	debugRaw: false,
```

Comment:
I don't like having more than one debug option. If this is specifically for debug, we should reuse `debug` (and change its description accordingly). If they are indeed unrelated, then this one should not be called `debugXXX`. Makes sense?